src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -73,11 +70,14 @@
         public static final boolean FIELD_CENTRIC_DEFAULT = true;
 
         public static final double TIP_P = 0.05, TIP_F = 0, TIP_TOLERANCE = 10, ACCEL_LIMIT = 0.0001;
+
+        public static final Rotation2 PRACTICE_BOT_DRIVE_OFFSET = Rotation2.fromDegrees(0), // should be 90
+                COMP_BOT_DRIVE_OFFSET = Rotation2.fromDegrees(0);
```

Comment:
I believe I heard talk of this being a 90 offset for camera oriented driving?  Or was this offset accounted for somewhere else?

Reply:
Yes, it is. New changes added the actual offset